presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -783,6 +789,11 @@ public static HiveCompressionCodec getTemporaryTableCompressionCodec(ConnectorSe
         return session.getProperty(TEMPORARY_TABLE_COMPRESSION_CODEC, HiveCompressionCodec.class);
     }
 
+    public static boolean isUsePageFileForHiveUnsupportedType(ConnectorSession session)
```

Comment:
```suggestion
    public static boolean shouldUsePageFileForHiveUnsupportedType(ConnectorSession session)
```
or
```suggestion
    public static boolean usePageFileForHiveUnsupportedType(ConnectorSession session)
```

Reply:
I initially use `usePageFileForHiveUnsupportedType` too, then I realized most other functions started with `is` as prefix for boolean type.  For example `isUseParquetColumnNames`